tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -103,6 +103,32 @@ public void EnsureContainerWithArgsEmitsContainerArgs()
             arg => Assert.Equal("more", arg.GetString()));
     }
 
+    [Fact]
+    public void EnsureExecutableWithArgsEmitsExecutableArgs()
```

Comment:
Perhaps add a variant that ensures args are added via the extension method even if the resource itself had no args added.

Reply:
Done.